### ISSUE TITLE
shasum fails when running npm install from Jenkins CI

### ISSUE THREAD

**Issue by duaneallam** [npm/npm#9915]
Hi,

I have a project set up on Jenkins CI build server on AWS and when I try and build the latest version I am getting shasum failures, seemingly limited to 2 packages: im-resize and im-metadata, which are dependencies of the s3-uploader package.

I'm unsure if this is a problem with npm, Jenkins or the packages themselves... has anyone else encountered similar issues?

Here is the tail end of my npm-debug.log file I can get more if needed:

34525 verbose stack Error: shasum check failed for /tmp/npm-10992-1c346078/registry.npmjs.org/im-resize/-/im-resize-2.3.1.tgz
34525 verbose stack Expected: d9aa5ecc6f39b893afc3e95bffaca7bc157a0848
34525 verbose stack Actual:   f47da25fb47704ba5c7e9f598bed81c676c1e0dc
34525 verbose stack From:     https://registry.npmjs.org/im-resize/-/im-resize-2.3.1.tgz
34525 verbose stack     at /var/lib/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/Nodejs/lib/node_modules/npm/node_modules/sha/index.js:38:8
34525 verbose stack     at ReadStream.<anonymous> (/var/lib/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/Nodejs/lib/node_modules/npm/node_modules/sha/index.js:85:7)
34525 verbose stack     at ReadStream.emit (events.js:129:20)
34525 verbose stack     at _stream_readable.js:908:16
34525 verbose stack     at process._tickCallback (node.js:355:11)
34526 verbose cwd /var/lib/jenkins/jobs/Portal Dev/workspace
34527 error Linux 3.13.0-49-generic
34528 error argv "/var/lib/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/Nodejs/bin/node" "/var/lib/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/Nodejs/bin/npm" "install"
34529 error node v0.12.7
34530 error npm  v2.11.3
34531 error shasum check failed for /tmp/npm-10992-1c346078/registry.npmjs.org/im-resize/-/im-resize-2.3.1.tgz
34531 error Expected: d9aa5ecc6f39b893afc3e95bffaca7bc157a0848
34531 error Actual:   f47da25fb47704ba5c7e9f598bed81c676c1e0dc
34531 error From:     https://registry.npmjs.org/im-resize/-/im-resize-2.3.1.tgz
34532 error If you need help, you may report this error at:
34532 error     <https://github.com/npm/npm/issues>
34533 verbose exit [ 1, true ]
34534 verbose unbuild node_modules/s3-uploader
34535 info preuninstall s3-uploader@1.0.1
34536 info uninstall s3-uploader@1.0.1
34537 verbose unbuild rmStuff s3-uploader@1.0.1 from /var/lib/jenkins/jobs/Portal Dev/workspace/node_modules
34538 info postuninstall s3-uploader@1.0.1
34539 silly gentlyRm /var/lib/jenkins/jobs/Portal Dev/workspace/node_modules/s3-uploader is being purged from base /var/lib/jenkins/jobs/Portal Dev/workspace
34540 verbose gentlyRm don't care about contents; nuking /var/lib/jenkins/jobs/Portal Dev/workspace/node_modules/s3-uploader
34541 silly vacuum-fs purging /var/lib/jenkins/jobs/Portal Dev/workspace/node_modules/s3-uploader
34542 silly vacuum-fs quitting because other entries in /var/lib/jenkins/jobs/Portal Dev/workspace/node_modules

**Comment by KenanY** [comment]
@duaneallam Are you still seeing this issue?

**Comment by duaneallam** [comment]
I don't know... Its been almost a year since I encountered this issue and I've since moved on to a different company and project.

I recall coming up with a workaround, not sure what it was exactly, either rolling NPM back to a previous version or setting the versions for im-resize and im-metadata in package.json or something else to that effect.

Are you getting the same issue or just doing some housecleaning?  If its the latter then feel free to close the issue...
